C/Coin-OR/CSDP/build_tarballs.jl

```diff
@@ -45,6 +45,12 @@ cp /usr/local/bin/csdp ${bindir}/csdp
 if [[ "${target}" == *-mingw* ]]; then
     mv "${bindir}/csdp" "${bindir}/csdp${exeext}"
 fi
+
+cd lib
+ar x libsdp.a
+cc -shared -o "libcsdp.${dlext}" *.o
+rm *.o
+cp libcsdp.${dlext} ${libdir}/libcsdp.${dlext}
```

Comment:
Don't copy, move instead, otherwise the library would be both in `lib/` and `bin/` and Windows.  Or more simply just create the shared library in `${libdir}`:
```suggestion
cc -shared -o "${libdir}/libcsdp.${dlext}" *.o
rm *.o
```